### PR TITLE
D3

### PR DESCRIPTION
**Pull request by valerii-fed**
Created by Reviewer Bot

### CODE REVIEW

**Comment by valerii-fed** [comment]
Created by: valerii.fedorenko.ua@gmail.com
Time: 2018-10-15T19:24:12Z
PR comment

File: file.txt [hunk 1]
```diff
@@ -27,4 +24,14 @@ line 26
 line 27
 line 28
 line 29
-line 30
+
+
+1
+2
+3
+4
```
Comment (valerii-fed) [inline file.txt]: Created by: valerii.fedorenko.ua@gmail.com
Time: 2018-10-06T11:50:23Z
Body: 32 on right
See in Reviewer: https://startupos-5f279.firebaseapp.com/diff/54/test-repo2/file.txt

File: file.txt [hunk 2]
```diff
@@ -2,22 +2,19 @@ line 1
...
 line 9
-line 10
 line 11
 line 12
+4444444444444444444444444444444444444
 line 13
 line 14
-line 15
 line 16
 line 17
```
Comment (valerii-fed) [inline file.txt]: Created by: valerii.fedorenko.ua@gmail.com
Time: 2018-10-06T11:50:23Z
Body: 15 on right
See in Reviewer: https://startupos-5f279.firebaseapp.com/diff/54/test-repo2/file.txt

File: file.txt [hunk 3]
```diff
@@ -2,22 +2,19 @@ line 1
 line 2
 line 3
 line 4
-line 5
 line 6
```
Comment (valerii-fed) [inline file.txt]: Created by: valerii.fedorenko.ua@gmail.com
Time: 2018-10-06T17:14:28Z
Body: 6 white
See in Reviewer: https://startupos-5f279.firebaseapp.com/diff/54/test-repo2/file.txt

File: file.txt [hunk 4]
```diff
@@ -2,22 +2,19 @@ line 1
...
 line 7
 line 8
 line 9
-line 10
 line 11
 line 12
+4444444444444444444444444444444444444
```
Comment (valerii-fed) [inline file.txt]: Created by: valerii.fedorenko.ua@gmail.com
Time: 2018-10-06T17:14:44Z
Body: 11 green
See in Reviewer: https://startupos-5f279.firebaseapp.com/diff/54/test-repo2/file.txt

File: file.txt [hunk 5]
```diff
@@ -2,22 +2,19 @@ line 1
...
 line 13
```
Comment (valerii-fed) [inline file.txt]: Created by: valerii.fedorenko.ua@gmail.com
Time: 2018-10-06T21:06:25Z
Body: 13 left
See in Reviewer: https://startupos-5f279.firebaseapp.com/diff/54/test-repo2/file.txt

File: file.txt [hunk 6]
```diff
@@ -27,4 +24,14 @@ line 26
 line 27
 line 28
```
Comment (valerii-fed) [inline file.txt]: Created by: valerii.fedorenko.ua@gmail.com
Time: 2018-10-06T22:13:48Z
Body: 25 right

See in Reviewer: https://startupos-5f279.firebaseapp.com/diff/54/test-repo2/file.txt

**Comment by val-fed** [comment]
First comment

**Comment by val-fed** [comment]
> First comment

Second comment

**Pull request closed without merging**